mobly/utils.py

```diff
@@ -291,8 +290,8 @@ def concurrent_exec(func, param_list, max_workers=30, raise_on_exception=False):
       try:
         return_vals.append(future.result())
       except Exception as exc:
-        logging.exception("{} generated an exception: {}".format(
-            params, traceback.format_exc()))
+        logging.exception('%s generated an exception: %s', params,
+                          traceback.format_exc())
```

Comment:
can you double check to see if the `traceback.format_exc()` is still needed in py3?

Reply:
Should probably fix in a separate PR. Let's merge this one :)

Reply:
I think this is still needed:
https://docs.python.org/3/library/traceback.html#traceback.format_exc